### ISSUE TITLE
Ein Link zu "Melden Sie sich beim Admin Center für Microsoft Teams an."

### ISSUE THREAD

**Issue by aisbergde** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#2889]
Hier wäre ein Link nützlich, wo beschrieben wird, wo und wie man dieses "Admin Center" findet.
Gibt es das auch in der kostenlosen Teams-Version?
Wenn nein, wie kann man in dieser den Gastzugriff aktivieren?

---
#### Dokumentdetails

⚠ *Bearbeiten Sie diesen Abschnitt nicht. Er ist für die Verknüpfung von docs.microsoft.com zum GitHub-Artikel erforderlich.*

* ID: 042d7ce6-ee22-690f-8188-ff8a4a585847
* Version Independent ID: 3e5bc99c-e08e-44e0-cda3-687ed3633175
* Content: [Aktivieren oder Deaktivieren des Gastzugriffs auf Microsoft Teams](https://docs.microsoft.com/de-de/microsoftteams/set-up-guests)
* Content Source: [Teams/set-up-guests.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/set-up-guests.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by OfficeGlobal** [comment]
**Bing translator:** "Here would be useful a link where to describe, where and how to find this "Admin Center".
Is this also available in the free Team version?
If not, how can You Enable Guest Access in this one?"
___
@scanum It seems the question is for a writer not for a loc. team

**Comment by lindspea** [comment]
@aisbergde https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2887

**Comment by aisbergde** [comment]
yes, it is for the writer. There should a link like https://admin.teams.microsoft.com/ 

and if a login is not possible because the person is not an admin or if login is possible the there are no enough rights, then some information is required who is admin to be able to contact this person. But this is not a question of documentation, this needs to be implemented in Teams or on the website to make it more user friendly.

**Comment by lindspea** [comment]
@aisbergde Have a look at these articles and let me know if they help at all - https://docs.microsoft.com/microsoftteams/manage-freemium and https://support.office.com/article/manage-your-teams-free-organization-5208c55a-5d66-416c-b99a-1028c3fef637

**Comment by aisbergde** [comment]
I have been on this sites last week and I wrote some feedback on this sites. The issue is: the idea is a little bit clear but there is not enough informations on this sites to get a full understanding. There is no definition for the item "organization", there is no information how to find out who is the "first person of an organzation" which has limited admin rights. And there is no link in the documentation to https://admin.teams.microsoft.com/ 
BTW this typical for many microsoft documentations: there is a lot of information, a normal user can think, all is well documented, but the final and importand links are often missing. And then a normal user needs to search in the internet. And an answer can be find outside the Microsoft documentation.
And even if I spent my time to give you feedback, what is missing in the documentation to make it better to understand, and you could use this feedback to improve the documentation, I get links, which I already visited to write again and again, what is missing. It is a little bit "the castel" from Kafka 
I am really interested in helping to improve the documentation and to make it better to understand.

**Comment by scanum** [comment]
@aisbergde Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by lindspea** [comment]
@aisbergde it seems that with the free version there is no access to the admin center. You will have to upgrade to the full version for this. For more information about the Teams Free version including organizations there is also a FAQ section on the app.  To enable guest access please see here https://support.office.com/article/add-guests-to-a-team-in-teams-fccb4fa6-f864-4508-bdde-256e7384a14f?ui=en-US&rs=en-US&ad=US. I have tested it and it works well.

**Comment by aisbergde** [comment]
I tested the free version and I got access to some kind of administration after I found the link
'https://admin.teams.microsoft.com'
I tried several emails of my domain @aisberg.de (which is used as the mysterious "organization" as I found out later, but of course this is not described in any place in the teams documentation) and finally I found one email which has Admin rights in this "organization". 
The usage of teams would be much easier if the "organization" would be shown by name directly in teams and also the email of the administrator. But teams prefer to hide important information in the app. 
Also Microsoft is not willing to add the link into the documentation and so also other user will not find the link by reading Microsoft documentation but only by using search engines.

I am often impressed how many resources Microsoft invest in generating a lot of very good documentation and how often the very last step in the documentation is missing.
